package.json

```diff
@@ -64,13 +63,11 @@
   "scripts": {
     "fix": "run-p fix:*",
     "fix:content": "remark . --output --ext md --quiet && prettier --write '**/*.md' --embedded-language-formatting off --loglevel silent",
-    "fix:json": "npm run lint:json --fix",
```

Comment:
I think we can also remove this stuff on lines 47 thru 49, it's part of the json linting stuff
```
  "**/*.json": [
    "npx eslint --fix --config '.eslintrc.js'"
  ],
```